src/ui/window/TopWindow.hpp

```diff
@@ -139,8 +140,8 @@ class TopWindow : public ContainerWindow {
 #ifdef DRAW_MOUSE_CURSOR
   uint8_t cursor_size = 1;
   bool invert_cursor_colors = false;
+  BrokenDateTime cursor_visible_until = BrokenDateTime::NowUTC();
```

Comment:
This should be `std::chrono::steady_clock::time_point` and not `BrokenDateTime`.
- `BrokenDateTime` is a very complicated structure that should only be used when you really need it broken down - the nice C++ class with its overloaded operators hides all the complexity for you, but it's there.
- `BrokenDateTime` is a system clock, not a monotonic clock. System clocks should only be used when there's a reason to; a monotonic ("steady") clock is safe against time warps and should always be preferred.

Reply:
I simply searched and reused the code was already there on the code. First thing I found was BrokenDateTime so I never knew about the other time related code you mention. But if it is better then I will look into it and will adjust it to that.

Reply:
Done. 
I changed the usage of `BrokenDateTime` to `std::chrono::steady_clock::time_point` and removed the previously added include for `BrokenDateTime`.